test/compression.js

```diff
@@ -661,6 +699,70 @@ function createServer (opts, fn) {
   })
 }
 
+function createHttp2Server (opts, fn) {
+  var _compression = compression(opts)
+  var server = http2.createServer(function (req, res) {
+    req.on('error', function (error) {
+      console.error('An error event occurred on a http2 request.', error)
```

Comment:
No one is going to comb through the CI logs after every run or upgrade... do these need to be here vs just letting an error crash the process? If they are preventing an expected error, it shouldn't be logging to the screen, as there should be no extra output when everything was actually OK.